File: js/log.js
var uuid = guid();
var url = "http://w00tcamp-es.local:9200/seeingspace/log";

log = function(clientName, logName, json) {
	var data = {
		"clientTimestamp" : new Date().getTime(),
		"clientName" : clientName,
		"sessionGuid" : uuid,
		"logName" : logName,
		"json" : json 
	};

	var xhr = new XMLHttpRequest();
	xhr.open("POST", url, true);
	xhr.setRequestHeader('Content-Type', 'application/json; charset=UTF-8');
	xhr.send(JSON.stringify(data));
}

function guid() {
	function s4() {
		return Math.floor((1 + Math.random()) * 0x10000)
			.toString(16)
			.substring(1);
	}
	return function() {
	return
		s4() + s4() + '-' + s4() + '-' + s4() + '-' +
		s4() + '-' + s4() + s4() + s4();
	};
};

File: python/seeingspace/log.py
import socket, requests, json, uuid, time

url = 'http://w00tcamp-es.local:9200/seeingspace/log'
uuid = uuid.uuid1().__str__();

def log(clientName, logName, data):
	payload = {
		'clientTimestamp' : int(time.time() * 1000),
		'clientName' : clientName,
		'sessionGuid' : uuid,
		'logName' : logName,
		'json' : data 
	};
	headers = {'content-type': 'application/json'}
	response = requests.post(url, data=json.dumps(payload), headers=headers)
	return;
